bindings/zeebe/jobworker/jobworker.go

```diff
@@ -104,27 +106,31 @@ func (z *ZeebeJobWorker) Read(ctx context.Context, handler bindings.Handler) err
 		return fmt.Errorf("binding is closed")
 	}
 
+	var retryBackOff *time.Duration
+	if z.metadata.RetryBackOff.Duration != time.Duration(0) {
+		retryBackOff = &z.metadata.RetryBackOff.Duration
+	}
+
 	h := jobHandler{
 		callback:     handler,
 		logger:       z.logger,
 		ctx:          ctx,
 		autocomplete: z.metadata.Autocomplete == nil || *z.metadata.Autocomplete,
+		retryBackOff: retryBackOff,
 	}
 
 	jobWorker := z.getJobWorker(h)
 
 	z.wg.Add(1)
 	go func() {
 		defer z.wg.Done()
-
+		// Wait for context cancelation or closure.
 		select {
-		case <-z.closeCh:
 		case <-ctx.Done():
+		case <-z.closeCh:
 		}
 
 		jobWorker.Close()
-		jobWorker.AwaitClose()
```

Comment:
Why was this removed?

Reply:
Close uses AwaitClose internally: https://github.com/camunda/zeebe/blob/71d2e6accc9f0452c72c1de7c16f18cc24d22552/clients/go/pkg/worker/jobWorker.go#L48